src/components/structures/FilePanel.js

```diff
@@ -190,6 +208,26 @@ const FilePanel = createReactClass({
         }
     },
 
+    _getScrollState: function() {
+        const messagePanel = this._messagePanel;
```

Comment:
Let's call this the `timelinePanel` throughout to avoid confusion when browsing the code.